src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -15,6 +15,8 @@ public class SearchDictionary<V>
         protected readonly Dictionary<V, Dictionary<string, double>> entryDictionary =
             new Dictionary<V, Dictionary<string, double>>();
 
+        public bool experimentalSearch = false;
```

Comment:
can this be made private?

Reply:
one option because this is an internal API is mark this internal and use [internalsVisibleAttribute] to make this property accessible from DynamoCoreWPF assembly.